deploy/templates/nstemplatetiers/appstudio/ns_tenant.yaml

```diff
@@ -156,7 +155,6 @@ objects:
     limits:
     - type: "Container"
       default:
-        cpu: 2000m
```

Comment:
If we want to remove the CPU limit from the LimitRange that is namespace wide, I think we need to remove cpu limits from all quotas, not only compute-build

Reply:
Yes, we also would need to remove it from the NotTerminating quota too.
Also, could you test that dropping the CPU limit only from the quotas and the limitrange and keeping the requests there actually works?

Reply:
Yeah.  As a first step can you give this one an ok-to-test to see what the automated tests do with "0" values, and we'll discuss how else we might need to test it in our next Infra team stand-up?

Reply:
For testing I would just:
- take a clean dev cluster (using cluster bot for example)
- create a namespace and create the same quota and limit range as in this PR
- try to create a Pod
If it's created and scheduled successfully then we are good IMO.

Unfortunately our (sandbox) e2e tests to do not currently tests tiers by creating pods.